testing/trino-product-tests-launcher/src/main/java/io/trino/tests/product/launcher/env/Environment.java

```diff
@@ -657,18 +657,19 @@ private void addConfiguredFeaturesConfig()
             DockerContainer testContainer = containers.get(TESTS);
             // write a custom tempto config with list of connectors the environment declares to have configured
             // since it's needed in TestConfiguredFeatures
-            ObjectMapper objectMapper = new ObjectMapper(new YAMLFactory());
             File tempFile;
             try {
                 tempFile = File.createTempFile("tempto-configured-features-", ".yaml");
-                objectMapper.writeValue(tempFile,
-                        Map.of("databases",
-                                Map.of("presto",
-                                        Map.of(
-                                                "configured_connectors",
-                                                configuredFeatures.asMap().getOrDefault(CONNECTOR, ImmutableList.of()),
-                                                "configured_password_authenticators",
-                                                configuredFeatures.asMap().getOrDefault(PASSWORD_AUTHENTICATOR, ImmutableList.of())))));
+                try (PrintWriter writer = new PrintWriter(tempFile)) {
+                    Yaml yaml = new Yaml();
+                    yaml.dump(Map.of("databases",
+                            Map.of("presto",
+                                    Map.of(
+                                            "configured_connectors",
+                                            configuredFeatures.asMap().getOrDefault(CONNECTOR, ImmutableList.of()),
+                                            "configured_password_authenticators",
+                                            configuredFeatures.asMap().getOrDefault(PASSWORD_AUTHENTICATOR, ImmutableList.of())))), writer);
```

Comment:
Super nit: can you move the `writer` to the next line? It's hard to spot that `yaml.dump()` actually takes two arguments.

Reply:
This particular instance didn't show up in the scan results - the instance that showed up was in warp speed (still not in executable codepath), being fixed in another PR.   Since it's a high-scoring critical finding we're being proactive about removing it everywhere, even if not in the executable path.